sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -1022,6 +1022,16 @@ public static void endSession() {
     return getCurrentScopes().startTransaction(transactionContext, transactionOptions);
   }
 
+  /** Starts the continuous profiler, if enabled. */
+  public static void startProfiler() {
```

Comment:
wdyt of introducing a subclass (like a namespace) `Sentry.profiling().start()` akin to `Sentry.metrics()`? We're also gonna do that for replay soon

Reply:
We want to keep the SDKs aligned. Other SDKs, like cocoa, is doing `SentrySDK.startProfiler`